gluten-core/src/main/scala/org/apache/gluten/backendsapi/TransformerApi.scala

```diff
@@ -70,4 +70,6 @@ trait TransformerApi {
   def getNativePlanString(substraitPlan: Array[Byte], details: Boolean): String
 
   def packPBMessage(message: Message): Any
+
+  def invalidateSQLExecutionResource(executionId: String): Unit = {}
```

Comment:
nit

Probably add a comment saying that the method is only for testing so far?

Reply:
addressed